src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -139,6 +139,8 @@ public function start(): bool
          * - The length of Windows and Linux filenames is limited to 255 bytes. Then the max must not exceed 255.
          * - The session filename prefix is `sess_`, a 5 bytes string. Then the max must not exceed 255 - 5 = 250.
          *
+         * This is @deprecated since Symfony 7.2 and only sid length of 32 will be allowed in Symfony 8.0.
```

Comment:
Deprecation is noted here, the regex will change not earlier than 8.0. It should be simplified to `/^[a-v0-9]{32}$/`. We can't really throw a deprecation here because there's an existing mechanism that regenerates a session id if it is detected invalid.